manifests/s/Seafile/Seafile/8.0.3/Seafile.Seafile.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Seafile.Seafile
+PackageVersion: 8.0.3
+PackageLocale: en-US
+Publisher: Seafile Ltd.
```

Comment:
```suggestion
Publisher: HaiWenHuZhi ltd.
```

Reply:
I went with the publisher/organization name listed on the [Seafile Website](https://www.seafile.com/en/about/#about-about), since I couldn't find any solid references to HaiWenHuZhi on the net, and Seafile Ltd. seems to be more prominent/easily found.

If you prefer the other name, let me know once more and I'll commit your change :) Thanks

Reply:
`Publisher` has to match the one shown in Control Panel for the best upgrade experience.

Reply:
Ahh, that's good to know for future packages, I'll keep it in mind.